normalizer/tests/test_https_endpoint.py

```
@@ -102,71 +102,59 @@ def test_https_availability():
-                    assert successful_responses > 0, f"テストしたエンドポイント {endpoints_tested} 個のうち、1個も応答しませんでした"
-        
-    except AssertionError as e:
-        print(f"アサーションエラー: {e}")
-        test_passed = False
-    except Exception as e:
-        print(f"予期しないエラー: {e}")
-        test_passed = False
-    
-    return test_passed
+    """統合テスト用のエントリーポイント"""
+    # 環境変数からベースURLを取得（未設定の場合はスキップ）
```

Ruff (0.14.13)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)